pkg/volume/flocker/client.go

```diff
@@ -0,0 +1,382 @@
+/*
```

Comment:
Is there anything kubernetes specific in this file? If not, I feel like this should be a separate project (a flocker go client) that anyone can reference as a dependency.